vendor/k8s.io/kubernetes/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1549,6 +1549,14 @@ func isPodBestEffort(pod *v1.Pod) bool {
 	return v1qos.GetPodQOS(pod) == v1.PodQOSBestEffort
 }
 
+// Check to see if node spec is set to Schedulable or not
+func CheckNodeUnschedulable(pod *v1.Pod, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
```

Comment:
let move this func to `pkg/scheduler/plugins/predicates/predicates.go`; we do not modify files in vendor directory :)